src/Command/AssertBackwardsCompatible.php

```diff
@@ -175,7 +181,8 @@ public function execute(InputInterface $input, OutputInterface $output) : int
             (new SymfonyConsoleTextFormatter($stdErr))->write($changes);
 
             $outputFormats = $input->getOption('format') ?: [];
-            Assert::that($outputFormats)->isArray();
+
+            assert(is_array($outputFormats));
```

Comment:
Same about `assert`...

Reply:
Similar to the above: will keep using `assert()` due to how the command is configured. If this keeps spawning everywhere, we'll need a psalm plugin.